client/js/s3/request-signer.js

```diff
@@ -265,7 +268,9 @@ qq.s3.RequestSigner = function(o) {
             pendingSignatureData = pendingSignatures[id],
             promise = pendingSignatureData.promise,
             signatureConstructor = pendingSignatureData.signatureConstructor,
-            errorMessage, response;
+            oldCredentials = Object.assign({}, credentialsProvider.get()),
```

Comment:
This will have to be removed/changed. `Object.assign` is not supported in any version of IE. Also, Safari, Chrome, and Firefox didn't add support until recently.